infrastructure/zk/src/hyperchain_wizard.ts

```diff
@@ -672,9 +668,26 @@ async function _generateDockerImages(_orgName?: string) {
         }
 
         if (process.env.PROVER_TYPE === ProverType.GPU) {
-            throw new Error('GPU prover configuration not available yet');
+            // throw new Error('GPU prover configuration not available yet');
+            hasGPUProver = true;
+            const cudaArchPrompt: BasePromptOptions[] = [
+                {
+                    message:
+                        'What is your GPU CUDA version? You can find it in table here - https://en.wikipedia.org/wiki/CUDA#GPUs_supported Input only 2 numbers withous dot, e.g. if you have RTX 3070 -> CUDA 8.6 -> Answer is 86',
```

Comment:
This is not correct, CUDA version and Compute Capability are not the same thing, here we want to know the Compute Capability of the GPU.